### DISCUSSION TITLE
C# Language Mind map

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#46] [General]
# C# Language Mind map

This blog post contains a mind map of language features starting from C# 1 up til now - including some of the new C# 12 features that will be released in November 2023.

https://steven-giesel.com/blogPost/f17d2762-b184-474b-9565-d5d40bf7cf49

**Comment by egil** [comment]
This is awesome. It would be even cooler if you made each feature clickble on the mind map, where the link takes you to the docs for the feature.

**Reply by linkdotnet** [reply]
Great idea - and yes that should be possible. At least in the PDF version.

**Reply by egil** [reply]
You can also have links in an svg, so that should work too.

**Reply by linkdotnet** [reply]
That's true - good point, I will work on that.

**Reply by linkdotnet** [reply]
@egil thanks again for the input - I re-created the mind mal as an SVG where all features are linked to the corresponding documentation!

**Reply by egil** [reply]
I like the font in the original better thought. The font in the svg is a little hard to read.

**Reply by linkdotnet** [reply]
I updated the font to be more readable - it differs a bit from the original png, but should be clearer to read
